commonknowledge/meep-intelligence-hub
# Add forgotten password functionality

## Description
This PR adds 
- two forms for requesting a password reset and resetting password with emailed token
- two new mutations for requesting password update and password reset
- custom email template for password reset
- a loading indicator on login button 

## Motivation and Context
Addresses issue [MEEP-311](https://linear.app/commonknowledge/issue/MEEP-311/forgot-password-functionality)

## How Can It Be Tested?
Download the branch and run locally
Navigate to http://localhost:3000/login and click on the button for Request Password Reset
Enter your account email 
Observe an email logged to the console
Click on the link in the email and fill out new password
Login with new password

## How Will This Be Deployed?
Usual deployment. Would be good to test when on production that env variables for sending email are working correctly

## Types of changes
